### DISCUSSION TITLE
Trascr_013: indicazioni "autorità mittente - "atto da trascrivere"

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1035] [Supporto servizi cooperativi]
Il caso d'uso
_Trascr_013 - Trascrizione di nascita: Trascrizione della sentenza di adozione di minore emessa dal Tribunale dei minori_
prevede la presenza della sezione "obbligatoria" della sezione "Autorità mittente":

![image](https://github.com/user-attachments/assets/42748f0f-2d14-4fd0-984e-4eaa333ca03a)

E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al Tribunale dei minori che abbia emesso la sentenza di adozione?

Inoltre, si prevede la presenza della sezione "obbligatoria" della sezione "Atto da Trascrivere":
![image](https://github.com/user-attachments/assets/c9d4983a-8206-43db-bb37-c1cacbd40105)

Nella situazione specifica, l'usc dovrebbe occuparsi della trascrizione della sola sentenza di adozione emessa dal Tribunale; è corretto dunque ipotizzare he qui debbano essere sempre riportate le informazioni relative alla sentenza emessa dal Tribunale medesmo? In dettaglio, è corretto che:
- il campo "Data formazione documento" riporti la data di emissione della sentenza di adozione?
- il campo "Estremi documento" riporti l'identificativo della sentenza di adozione?
- il campo "Nome Ente" riporti sempre l'indicazione (fissa) "Tribunale"?

**Comment by acolosimosogei** [comment]
Salve, abbiamo preso in carico la discussion #1035, le daremo un riscontro quanto prima.
Grazie.

**Comment by acolosimosogei** [comment]
Salve, in seguito riporto le domande proposte nella discussion #1035, seguite rispettivamente dalla risposta:

1) E' corretto ipotizzare che qui debbano essere sempre riportate le informazioni relative al Tribunale dei minori che abbia emesso la sentenza di adozione?
	E' corretto.

2) è corretto dunque ipotizzare he qui debbano essere sempre riportate le informazioni relative alla sentenza emessa dal Tribunale medesmo?
	E' corretto.

3) il campo "Data formazione documento" riporti la data di emissione della sentenza di adozione?
	E' corretto.

4) il campo "Estremi documento" riporti l'identificativo della sentenza di adozione?
	E' corretto.

5) il campo "Nome Ente" riporti sempre l'indicazione (fissa) "Tribunale"?
	Il campo nome ente viene lasciato "libero" in modo tale che l'ufficiale di stato civile possa valorizzare il campo con il valore che ritiene più consono.

Chiudiamo pertanto la discussion: in caso di ulteriori richieste si prega di aprirne un'altra dedicata.
Grazie
